code/game/objects/items/stacks/wrap.dm

```diff
@@ -203,3 +210,9 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_speed = 3
 	throw_range = 5
+
+/obj/item/c_tube/attack(mob/living/target_mob, mob/living/user, params)
+	. = ..()
+	visible_message("[user] baps [target_mob] on the head with the [src]")
```

Comment:
span_warning maybe, no spans doesn't make that much sense here, even if it's non-harmful